src/components/Input.jsx

```diff
@@ -1,14 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/bookSlice';
+import AddButton from './AddButton';
 
-const Input = () => (
-  <div>
-    <h2>Add new book</h2>
-    <form>
-      <input type="text" name="" id="" placeholder="Book Title" />
-      <input type="text" name="" id="" placeholder="Author" />
-      <button type="button">Add Book</button>
-    </form>
-  </div>
-);
+const Input = () => {
+  // <div>
+  // <h2>Add new book</h2>;
+  //   <form>
+  //     <input type="text" name="" id="" placeholder="Book Title" />
+  //     <input type="text" name="" id="" placeholder="Author" />
+  //     <button type="button">Add Book</button>
+  //   </form>
+  // </div>
```

Comment:
- You have done great so far :+1: :+1: Your redux store and project work as expected. But there are some best practice requirements you need to meet before moving to the next milestone. :+1: :100: 
- [ ] Following the best practices, you are not meant to leave unused code as a comment on your code base. I'll suggest you remove this as it makes your codebase look a bit a bit unprofessional. :+1: :smile: